### ISSUE TITLE
By request - latest.log for broken feature jsons that don't cause crash

### ISSUE THREAD

**Issue by bessiq** [TelepathicGrunt/Blame#1]
MC: 1.16.3
Forge: 34.1.1
Mods:
BetterCaves-1.16.2-1.0.2.jar
blame-forge-1.16.2-1.1.0
Biomes You Go 1.0.7 MCV: 1.16.3
Log: https://gist.github.com/bessiq/af68aa42141c34a28e732bbffb433ab1

Let me know if you need any other info or want me to test anything. I'm leaving Blame enabled _forever_ so if I come across anything I'll let you know. Thanks.

**Comment by bessiq** [comment]
So, I still have one "broken" feature with Better Caves disabled:
`[Render thread/ERROR] [net.minecraft.world.biome.BiomeGenerationSettings/]: Feature: Not a JSON object: "appliedenergistics2:quartz_ore"`

log: https://gist.github.com/bessiq/46acf26ff2b22ea0df45a840732bc083

The thing is, AE2 quartz ore _is_ generating normally, despite the error. However, this bug does prevent another mod, Large Ore Deposits, from generating that ore (I have a config for it to generate in specific biomes at a higher rate, which worked fine in 1.16.1).

The mod causing the issue in this case Good Night's Sleep (currently on v1.2.6). With only GNS and AE2 enabled, I get this feature error. With just AE2, there is no error.

Should I report this to the GNS dev, or the AE2 dev, or both? Or neither? I can live without the custom AE2 deposits from Large Ore Deposits, especially since I am starting to figure out how datapack worldgen works. Oh, could I maybe correct the issue by dropping that feature into a datapack myself? E.g.:
`data\appliedenergistics2\worldgen\configured_feature\quartz_ore.json`

Of course, I'd need to maybe see what the original version looks like, which I guess is generated by the alien-to-me code side of the mod. I could also maybe just base the config on a vanilla ore and give it a shot.

Anyway, sorry for rambling. And fwiw, I _did_ see Blame entries in my latest.log when I made some (unrelated) datapack worldgen errors that didn't result in a crash. They told me to notify you, but I figured it out!
